src/main/java/com/github/fishio/control/AchievementScreenController2.java

```diff
@@ -0,0 +1,102 @@
+package com.github.fishio.control;
+
+
+
+
+import com.github.fishio.Preloader;
+import com.github.fishio.achievements.Achievement;
+import com.github.fishio.logging.Log;
+import com.github.fishio.logging.LogLevel;
+
+import javafx.fxml.FXML;
+import javafx.scene.Scene;
+import javafx.scene.control.Button;
+import javafx.scene.control.Label;
+
+/**
+ * Achievement screen controller.
+ */
+public class AchievementScreenController2 implements ScreenController {
+	
+	private Log log = Log.getLogger();
+	private Achievement playerDeath = new Achievement("playerDeath");
+	private Achievement enemyKill = new Achievement("enemyKill");
+	private Achievement hitWall = new Achievement("HitWall");
+	private Achievement revive = new Achievement("Revive");
+	private Achievement pun = new Achievement("Pun");
+
+	@FXML
+	private Button btnBackToMenu;
+	private Label bigachieve1;
```

Comment:
I'm afraid all these fields need a ``@FXML`` to be useful. Besides, we don't really need them in the code, they are not used anyway.

<naming